documentation/core/Modifiers.md

```diff
@@ -0,0 +1,20 @@
+# Modifiers
+A modifier is: ...
+
+The main modifiers are `Attribute` and `Format`.
+
+## Attributes
+When parsing and rendering a document, we need to save the attributes that a
+DOM node has and save it in our VNode.
+
+Attributes exists to:
+- retrieve HTML attributes that a DOM node has originally
+- render the HTML attributes that a VNode has
+
+## Format
+The DOM inline nodes (e.g. span, strong, em, b, a, ...) in HTML are difficult to
+manipulate. We therefore we transform them into a Format modifiers that we
+attach to a VNode through the "modifiers" properties.
```

Comment:
Everywhere in the DOC please don't forget to use \`\` for the references to classes etc.
In this case: `VNode` instead of VNode.